example/DeltaBlue.dart

```diff
@@ -87,19 +88,16 @@ class Strength {
   }
 }
 
-
 // Compile time computed constants.
-const REQUIRED        = const Strength(0, "required");
+const REQUIRED = const Strength(0, "required");
```

Comment:
[nit] this could be updated to lowercase constants, but honestly -- let's just land this first.

Reply:
yup

Reply:
Please do not reformat benchmarks. It makes it harder to compare versions.

Reply:
Hrm...we standardize on dartfmt for everything – no one has touched this package in years...

Why is this important now?